src/option.c

```diff
@@ -149,7 +149,7 @@ void set_default_options(void)
    opt_set_str(OPT_EVAL_VERBOSE, getenv("NVC_EVAL_VERBOSE"));
    opt_set_str(OPT_ELAB_VERBOSE, getenv("NVC_ELAB_VERBOSE"));
    opt_set_size(OPT_HEAP_SIZE, 16 * 1024 * 1024);
-   opt_set_int(OPT_ERROR_LIMIT, 20);
+   opt_set_int(OPT_ERROR_LIMIT, 0);
```

Comment:
Thanks for reminding me about this, it's fixed as part of 63a8319f. It was only ever supposed to apply to analysis.